lib/SetupChecks/OcrMyPdfCheck.php

```diff
@@ -47,6 +49,9 @@ public function getName(): string {
 	}
 
 	public function run(): SetupResult {
+		if ($this->ocrBackendInfoService->isRemoteBackend()) {
+			return SetupResult::success($this->l10n->t('Workflow OCR Backend is installed.')); // TODO :: health check ?
```

Comment:
Does a health check have to be performed here to generate a `SetupResult::error` or `SetupResult::warning` object, or is a setup error already caught in the generic application logic of the remote setup?

Reply:
`heartbeat` check implemented